src/App.tsx

```diff
@@ -21,19 +21,26 @@ enum SortType {
   LENGTH,
 }
 
-// Use this function in the render method
 function getReorderedGoods(
   goods: string[],
   sortType: SortType,
-  isReversed: boolean,
 ) {
-  // Not to mutate the original array
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // ...
+  switch (sortType) {
+    case SortType.LENGTH:
+      return visibleGoods.sort((g1, g2) => {
+        return g1.length - g2.length;
+      });
+
+    case SortType.ALPABET:
+      return visibleGoods.sort((g1, g2) => {
+        return g1.localeCompare(g2);
+      });
 
```

Comment:
Switch should always have default case
```suggestion
  switch (sortType) {
    case SortType.LENGTH:
      return visibleGoods.sort((g1, g2) => g1.length - g2.length);

    case SortType.ALPABET:
      return visibleGoods.sort((g1, g2) => g1.localeCompare(g2));
      
    case SortType.NONE:
    default:
      break;

```